<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordResetWindow</class>
 <widget class="QMainWindow" name="PasswordResetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Reset</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background-color: #7B68EE;
    }
    QWidget#forgotPasswordForm, QWidget#newPasswordForm {
        background-color: white;
        border-radius: 12px;
    }
    QLabel#titleLabel {
        font-size: 20px;
        font-weight: bold;
        color: #333333;
        margin-bottom: 10px;
    }
    QLabel#subtitleLabel {
        font-size: 14px;
        color: #666666;
    }
    QLabel#messageLabel {
        background-color: #E8F5E9;
        padding: 10px;
        border-radius: 6px;
        color: #2E7D32;
        font-size: 13px;
    }
    QLineEdit {
        padding: 12px;
        border: 1px solid #E0E0E0;
        border-radius: 6px;
        font-size: 14px;
        min-height: 20px;
    }
    QPushButton {
        background-color: #e2b4b7;
        color: white;
        border: none;
        border-radius: 6px;
        padding: 12px;
        font-size: 14px;
        min-height: 45px;
    }
    QPushButton:hover {
        background-color: #bd637e;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="forgotPasswordPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="forgotPasswordForm" native="true">
          <layout class="QVBoxLayout" name="formLayout">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="leftMargin">
            <number>30</number>
           </property>
           <property name="rightMargin">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="titleLabel">
             <property name="text">
              <string>Forgot Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="subtitleLabel">
             <property name="text">
              <string>Nhập địa chỉ email của bạn.</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="emailInput">
             <property name="placeholderText">
              <string>Nhập địa chỉ email</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="continueButton">
             <property name="text">
              <string>Continue</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="newPasswordPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="newPasswordForm" native="true">
          <layout class="QVBoxLayout" name="formLayout_2">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="leftMargin">
            <number>30</number>
           </property>
           <property name="rightMargin">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="titleLabel_2">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>New Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="messageLabel">
             <property name="text">
              <string>Vui lòng tạo một mật khẩu mới 
khỏe và an toàn hơn.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="newPasswordInput">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Tạo mật khẩu mới</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="confirmPasswordInput">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Xác nhận mật khẩu của bạn</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="changeButton">
             <property name="text">
              <string>Change</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
